docs/cpp/faq-cpp.md

```diff
@@ -0,0 +1,79 @@
+
+1. Update your `includePath` (and preprocessor defines)
+2. Force semantic IntelliSense
+
+### Update your `includePath` (and preprocessor defines)
+
+Selecting this option will open a file called **c_cpp_properties.json**. If you haven't created this file already, it will be created for you in the **.vscode** folder of your workspace.
+
+Add the necessary paths to your include files to the `includePath` array. The `${workspaceRoot}` variable is available to use to get a relative path to the folder you have opened. Also add any required symbols that need to be defined to the `defines` array. Both "\<var\>" and "\<var\>=\<value\>" syntax is accepted. When you edit and save this file, the IntelliSense engine will reset and reparse source your source files and headers with the new settings.
+
+### Force semantic IntelliSense
+
+If you want IntelliSense to operate on your files even when all #include directives do not resolve, then you can choose the `Force semantic IntelliSense` code action to always use the new IntelliSense engine. You can also set the `C_Cpp.intelliSenseEngineFallback` setting to `Disabled`.
+
+## Why do I see red squiggles under Standard Library types?
+
+The most common reason for this is missing include paths and defines. The easiest way to fix this on each platform is as follows:
+
+**Linux/Mac**: Set `intelliSenseMode": "clang-x64` or `intelliSenseMode": "gcc-x64` and `compilerPath` in **c_cpp_properties.json** to the path to your compiler.
+
+**Windows**: If you are using the Microsoft C++ compiler, set `intelliSenseMode": "msvc-x64`, but don't add the `compilerPath` property to **c_cpp_properties.json**. If you are using Clang for Windows, set `intelliSenseMode": "msvc-x64`, and `compilerPath` in **c_cpp_properties.json** to the path to your compiler.
+
+## How do I get the new IntelliSense to work with MinGW on Windows?
+
+The page discussing configuration with MinGW is [here](https://github.com/Microsoft/vscode-cpptools/blob/master/Documentation/LanguageServer/MinGW.md).
+
+## How do I get the new IntelliSense to work with the Windows Subsystem for Linux?
+
+See [Get Started with C++ and Windows Subsystem for Linux in Visual Studio Code](config-wsl.md).
+
+## What is the difference between `includePath` and `browse.path` in **c_cpp_properties.json**?
+
+There are two settings in the **c_cpp_properties.json** file. They are used by the different IntelliSense engines that we support and have slightly different meanings for the components that use them.
```

Comment:
this section could use an update.  `browse.path` is a more "advanced" setting now and hardly anyone should be using it.

Reply:
I need some specifics.

Reply:
I think we should just delete the topic.

Reply:
done